lightkurve/tests/test_targetpixelfile.py

```diff
@@ -60,6 +60,41 @@ def test_tpf_shapes():
         assert tpf.quality_mask.shape == tpf.hdu[1].data['TIME'].shape
         assert tpf.flux.shape == tpf.flux_err.shape
 
+def test_tpf_math():
+    """Can you add, subtract, multiply and divide?"""
+    with warnings.catch_warnings():
+        # Ignore the "TELESCOP is not equal to TESS" warning
+        warnings.simplefilter("ignore", LightkurveWarning)
+        tpfs = [KeplerTargetPixelFile(filename_tpf_all_zeros),
+                TessTargetPixelFile(filename_tpf_all_zeros)]
+    # These should work
+    for tpf in tpfs:
+        for other in [1, np.ones(tpf.flux.shape[1:]), np.ones(tpf.shape)]:
+            tpf + other
+            tpf - other
+            tpf * other
+            tpf / other
+
+
+            tpf += other
+            tpf -= other
+            tpf *= other
+            tpf /= other
+
+    # These should fail with a value error because their shape is wrong.
+    for tpf in tpfs:
+        for other in [np.asarray([1, 2]), np.arange(len(tpf.time) - 1), np.ones([100, 1]), np.ones([1, 2, 3])]:
+            with pytest.raises(ValueError):
+                tpf + other
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4836899)